### PR TITLE
Add created and updated at into schemas, sort rooms by updatedAt

### PR DESCRIPTION
**Pull request by dbertella**

### CODE REVIEW

File: packages/server/src/database/models.ts [hunk 1]
```diff
@@ -5,37 +5,50 @@ import { GameState } from 'shared';
 export interface IPlayer extends mongoose.Document {
   name: string;
 }
-const PlayerSchema = new mongoose.Schema({
-  name: String,
-});
+const PlayerSchema = new mongoose.Schema(
+  {
+    name: String,
+    createdAt: Date,
```
Comment (stathismor) [inline packages/server/src/database/models.ts]: Are you sure the `createdAt` and `updateAt` are needed here? I think according to documentation, mongoose will add those. Maybe we need them in the interface, but not the schema.
https://mongoosejs.com/docs/guide.html#timestamps

File: packages/server/src/database/models.ts [hunk 2]
```diff
@@ -5,37 +5,50 @@ import { GameState } from 'shared';
...
+  {
+    timestamps: true,
+  },
+);
 
 export interface IRoom extends mongoose.Document {
   name: string;
   owner: string;
-  createdAt: string;
+  createdAt: Date;
```
Comment (stathismor) [inline packages/server/src/database/models.ts:22]: Should we add both of timestamps to this and player interfaces?

File: packages/server/src/database/models.ts (hunk 1 above)
Comment (dbertella) [reply packages/server/src/database/models.ts]: I'm not :-P how could I test this? Should I just remove them and see if everything works?

Comment (dbertella) [reply packages/server/src/database/models.ts]: done

**Review by stathismor — APPROVED** [review]

**Pull request merged**